ros/docker/encoder_odom/src/encoder_arduino.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
This file hasn't been used for testing yet. Needs to incorporate the changes from encoder_odom.py as well so probably shouldn't merge it.